server/middlewares/proxy.ts

```diff
@@ -0,0 +1,19 @@
+const envs = require("env-smart")
+// read environment variable
+const OS_HTTP_PROXY = envs.load().HTTP_PROXY
+const OS_HTTPS_PROXY = envs.load().HTTPS_PROXY
+const proxy = () => {
```

Comment:
`function getProxy () {`

Reply:
We have properly defined the function as you suggested.